shopyo/conftest.py

```diff
@@ -94,15 +102,19 @@ def db_session(db):
     options = dict(bind=connection, binds={})
     session = db.create_scoped_session(options=options)
 
-    db.session = session
-
+    yield db  # this is where the testing happens!
     yield session
```

Comment:
This is also messed up.  Correct one is here https://github.com/rehmanis/shopyo/blob/3afe2e14c978854239130cfeadea547b89582ff8/shopyo/conftest.py